src/ReadingPanel/index.js

```diff
@@ -0,0 +1,16 @@
+import EmailHeader from "./EmailHeader"
+import EmailBody from "./EmailBody"
+import './style.css'
+import CloseButton from "./CloseButton"
+
+const DisplayReadingPanel = (props) => {
```

Comment:
the folder is called `ReadingPanel`, the file is called `index.js` and the component is called `DisplayReadingPanel`. it would be better if they are all called `ReadingPanel`.